app.js

```diff
@@ -17,6 +18,12 @@ module.exports = function(config) {
         }
         process.exit(1);
     })
+    
+    if(typeof(config.isReadOnlyMode) !== "undefined")
+    {
+      isReadOnlyMode = config.isReadOnlyMode;
+    }
+    config.isReadOnlyMode = isReadOnlyMode;
```

Comment:
Will change it undefined check, rather than typeof. Will send a new pull with the changes.